### ISSUE TITLE
Seems uncannily similar to me…

### ISSUE THREAD

**Issue by nguyencomputing** [dkoh05/path_finding_algo#3]
https://www.youtube.com/watch?v=JtiK0DOeI4A&ab_channel=TechWithTim

**Comment by Gabr1sal** [comment]
it's exactly the same
I think he should add a reference or something

**Comment by dkoh05** [comment]
Yup, it is! Updated it so that you can see the origin of this project.

**Comment by nguyencomputing** [comment]
What's the value of copying code and putting it on your GitHub though? What value are you adding?
